src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="container">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="/"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="link"
+            data-qa="hover"
+          >
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
```

Comment:
The `href` attributes for the navigation links are set to `#`, which is typically used as a placeholder. Ensure these links point to actual sections or pages if they exist, to improve navigation and user experience.